adm_program/installation/db_scripts/preferences.php

```diff
@@ -16,6 +16,7 @@
     // System
     'enable_rss'                     => '1',
     'enable_auto_login'              => '1',
+    'enable_login_emailaddress'      => '0',
```

Comment:
for the new preferences we use a prefix with the preference area so `security_login_email_address_enabled` would be a better name